src/main/java/slimeknights/tconstruct/library/modifiers/Modifier.java

```diff
@@ -324,6 +330,41 @@ public void onInventoryTick(IModifierToolStack tool, int level, World world, Liv
   }
 
 
+  /**
+    * Called when this item is used when targeting a Block
+   * @param tool 
+   * @param level 
+   * @param stack 
+   * @param context        Full item use context
+   * @return  Action result
+   */
+  public ActionResultType onItemUse(ToolStack tool, int level, ItemStack stack, ItemUseContext context) {
+    return ActionResultType.PASS;
+  }
+  
+  public ActionResultType itemInteractionForEntity(ToolStack tool, int level, ItemStack stack, PlayerEntity playerIn, LivingEntity target, Hand hand) {
```

Comment:
Any reason this is `ToolStack`? Most other modifier hooks use `IModifierToolStack`. If one of the modifiers downstream requires `ToolStack`, we should update that.

Reason being, eventually I plan to split off `ToolStack` to allow non-multipart modifiable items.

Aside, do we need the item stack here? I assume thats for the shearing, arguably they could refetch the item stack using the player (its cheap), I want to discourage using the item stack directly when possible

Reply:
IForgeShearable. I guess i could switch it to player access, just feels kinda weird to be asking when the stack is right here.

Reply:
We just got it from player access in the same way earlier. Its mostly about keeping the API smaller, fewer things to mismatch